plugins/org.eclipse.sirius.diagram.sequence/src/org/eclipse/sirius/diagram/sequence/business/internal/layout/vertical/SequenceVerticalLayout.java

```diff
@@ -771,45 +771,38 @@ private boolean layoutUnconnectedLostMessageEnd() {
     private boolean layoutInteractionContainer() {
         boolean applied = false;
         Set<InteractionContainer> allInteractionContainers = this.sequenceDiagram.getAllInteractionContainers();
-        if (!allInteractionContainers.isEmpty() && getInteractionContainerBounds() != null) {
+        if (!allInteractionContainers.isEmpty()) {
```

Comment:
You might perform the look up in org.eclipse.sirius.diagram.sequence.business.internal.layout.vertical.SequenceVerticalLayout.init(boolean)

as done for other kind of elements